app/views/orders/index.html.erb

```diff
@@ -4,10 +4,12 @@
   <table>
     <tr>
       <th>Order Number</th>
+      <th></th>
```

Comment:
Do we want to re-order this table so that this column without the header is at the end? Just a thought because having these tags just to have a column seems a little weird.